content/tutorials/signing-transactions-with-webauthn/_dir.yml

```diff
@@ -0,0 +1,24 @@
+title: Signing Transactions With WebAuthn
+featured: true
+authors:
+  - name: MatterLabs
+    url: https://matter-labs.io
+    avatar: https://avatars.githubusercontent.com/u/42489169?s=200&v=4
```

Comment:
I actually think you should author this by name rather then MatterLabs. Only reason I suggest is that if I got to community code rn I see 99% of company written content. Would be encouraging for others to see individuals contributing, at least thats a theory haha.